packages/now-cli/src/util/projects/create-project.ts

```diff
@@ -0,0 +1,13 @@
+import Client from '../client';
+import { Project } from '../../types';
+
+export default async function createProject(
+  client: Client,
+  projectName: string
+) {
+  const project = await client.fetch<Project>('/v1/projects', {
```

Comment:
Is `v1` the latest version of this endpoint?

Reply:
I was following the documentation here https://vercel.com/docs/api#endpoints/projects/create-a-project